crates/re_viewport/src/space_view.rs

```diff
@@ -92,13 +99,22 @@ impl SpaceViewBlueprint {
         let mut contents = SpaceViewContents::default();
         contents.insert_entities_according_to_hierarchy(queries_entities, space_path);
 
+        let mut root_entity_properties = EntityProperties::default();
+
+        // better defaults for the time series space view
+        if space_view_class == TimeSeriesSpaceView::NAME {
+            root_entity_properties.visible_history.nanos = VisibleHistory::ALL;
+            root_entity_properties.visible_history.sequences = VisibleHistory::ALL;
+        }
+
```

Comment:
Spotlighting this rather ugly hack that implement better default for the timeseries space view.

Should support for "per-space-view-class property defaults" be factored in the future blueprint architecture?

Reply:
This needs an issue and an actual comment:
```suggestion
        // better defaults for the time series space view
        // TODO(#TBD): Should support for "per-space-view-class property defaults" be factored in
        // the future blueprint architecture?
        if space_view_class == TimeSeriesSpaceView::NAME {
            root_entity_properties.visible_history.nanos = VisibleHistory::ALL;
            root_entity_properties.visible_history.sequences = VisibleHistory::ALL;
        }